chapters/en/slides/module1/module1_31.Rmd

```diff
@@ -107,14 +104,58 @@ Notes: Script here.
 
 ---
 
+To make a `bar` plot using `altair`, we follow the steps below:
+
+1. We take the dataframe we wish to plot, in this case `df`.
+2. We create an `altair` plot object using `alt.chart()`.
+3. We can pass the dataframe we'd like to plot by using `alt.chart(df)`
+4. But what kind of plot do we want?! A bar chart in this case would work nicely so lets add `.mark_bar()` after that.
+5. Now that we have specified that we want a `bar` plot from our `df`, we need to specify what column. This is done using the `.encode(x='mfr', y='count()')`. The `count` here says that we would like to count up all the occurrences of the different manufacturers.
+6. Putting it all together, we have ```alt.chart(df).mark_bar().encode(x='mfr', y='count()')```. Lets see how this looks on the next slide.
+
+---
+
+```{python}
+# Note that the setting the width and height is for consistency.
+
+chart1 = alt.Chart(df, width=500, height=300).mark_bar().encode(
```

Comment:
You didnt discuss width and height?!! I would do the first  without this, and then teach them that it doesn't look great so lets change the dimensions and poof explain these.